contentful/README.md

```diff
@@ -0,0 +1,63 @@
+## Overview
+
+[Contentful][1] is a content (articles, photos, and videos) management platform that allows businesses to create, manage, and deliver digital content across various channels like websites and mobile apps through its intuitive interface and robust APIs or SDKs.
+
+Integrate with Datadog to gain insights into Contentful activities related to content and other actions as part of your Contenful spaces and environments.
+
+## Setup
+
+Follow the instructions below to configure this integration for your Contentful space.
+
+### Configuration
+
+#### Webhook Configuration
+Configure the Datadog endpoint to forward Contentful events as logs to Datadog. See [Contentful Webhook overview][2] for more details.
+
+1. Select an existing API key or create a new one by clicking one of the buttons below:<!-- UI Component to be added by DataDog team -->
+2. Log in to your [Contenful account][3] as space admin.
+3. Go to **Settings > Webhooks**.
+4. Click on **Add Webhook**.
+5. Add name and webhook URL generated from step 1.
```

Comment:
As far as I see, step 1 did not generate a webhook URL, so this step seems wrong.